common/src/main/scala/com/gu/zuora/model/SubscriptionData.scala

```diff
@@ -0,0 +1,31 @@
+package com.gu.zuora.model
+
+import org.joda.time.LocalDate
+
+case class SubscriptionData(ratePlanData: List[RatePlanData], subscription: Subscription)
+
+case class Subscription(
+  contractEffectiveDate: LocalDate,
+  contractAcceptanceDate: LocalDate,
+  termStartDate: LocalDate,
+  autoRenew: Boolean = true,
+  //scalastyle:off magic.number
+  initialTerm: Int = 12,
```

Comment:
You can add the `// scalastyle:off magic.number` to the end of the line that you want it off for rather than turning it on and off.
`initialTerm: Int = 12, //scalastyle:off magic.number`
`renewalTerm: Int = 12, //scalastyle:off magic.number`